src/test/java/uk/gov/pay/api/it/PaymentResourceSearchITest.java

```diff
@@ -570,6 +571,35 @@ public void searchPayments_withMandateId_whenCardPayment_shouldReturnABadRequest
                 .assertThat("$.description", is("Invalid parameters: agreement_id. See Public API documentation for the correct data formats"));
     }
 
+    @Test
+    public void searchPayments_ShouldIncludeCaptureLink_whenReturnedFromConnector() {
+        publicAuthMock.mapBearerTokenToAccountId(API_KEY, GATEWAY_ACCOUNT_ID);
+        String SUBMITTED_STATE = "submitted";
```

Comment:
This isn’t a constant, so it should really be in regular `camelCase` rather than `SCREAMING_SNAKE_CASE`